packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -34,10 +34,17 @@ void describe('getting started happy path', async () => {
     // start a local npm proxy and publish the current codebase to the proxy
     await execa('npm', ['run', 'clean:npm-proxy'], { stdio: 'inherit' });
     await execa('npm', ['run', 'vend'], { stdio: 'inherit' });
+    /**
+     * delete .bin folder in the repo because
+     * 1) e2e tests don't use them
+     * 2) execa would use them if it can not find the binary in the test project
```

Comment:
```suggestion
     * 1) e2e tests should not depend on them
     * 2) package managers will use them if it can not find a binary in the test project
```

Reply:
It's not 100% that the package manger uses it. I think it's more about `execa`

Reply:
Done eaf8440c1f